snuba/clickhouse/columns.py

```diff
@@ -40,7 +40,7 @@ def for_schema(self) -> str:
     @staticmethod
     def to_columns(
         columns: Sequence[Union[Column, Tuple[str, ColumnType]]]
-    ) -> List[Column]:
+    ) -> MutableSequence[Column]:
```

Comment:
Why does this need to be mutable?

Reply:
there was no reason, just did not think about changing that as well, fixed now.